scripts/integration

```diff
@@ -1,87 +0,0 @@
-#!/bin/bash
```

Comment:
Dropped this script entirely in favor of a package-ci script that is far less verbose and run per repository.

It doesn't make sense to run these tests in CI for charts-build-scripts if we are introducing breaking changes.